pages/learn/welcome/production-plan.md

```diff
@@ -118,6 +118,10 @@ When updating software on production devices, it makes sense to be cautious and
 
 The {{ $names.company.lower }} API includes functionality that allows for [fine-grained control of updates][staged-releases]. With these endpoints, you can turn off automatic release updates, set a target release for the whole fleet, or set target releases for a specific subset of devices. The [device tag][tags] and [filter][filters] functionality makes it easy to assign groups of devices to certain phases of your deployment.
 
+## Further reading
+
+For more information about common anti-patterns, see [How to break balenaOS (development anti-patterns)][anti-patterns].
```

Comment:
@garethtdavies i had to remove the templatized reference to balenaOS here as it broke the build, not sure if that's a known issue or not.

Reply:
Interesting and I've not seen that before. I actually built this locally with the variable syntax and it seemed to work `[How to break {{ $names.os.lower }} (development anti-patterns)][anti-patterns]`

Reply:
Well, I'll be, i don't know what i'm doing differently now but it's working fine. sorry for the trouble!